tests/autoblocks/test_tracer.py

```diff
@@ -448,3 +451,90 @@ def test_tracer_start_span(*args, **kwargs):
 
     assert tracer._properties.get("span_id") is None
     assert tracer._properties.get("parent_span_id") is None
+
+
+def test_tracer_prod_evaluations(httpx_mock):
+    test_evaluation_id = uuid.uuid4()
+
+    class MyEvaluator(BaseEventEvaluator):
+        id = "my-evaluator"
+
+        def evaluate_event(self, event: TracerEvent) -> EventEvaluation:
+            return EventEvaluation(
+                evaluator_external_id=self.id,
+                id=test_evaluation_id,
+                score=0.9,
+                threshold={"gte": 0.5},
+            )
+
+    mock_input = {
+        "trace_id": "my-trace-id",
+        "timestamp": timestamp,
+        "properties": {},
+        "evaluators": [
+            MyEvaluator(),
+        ],
+    }
+    httpx_mock.add_response(
+        url=INGESTION_ENDPOINT,
+        method="POST",
+        status_code=200,
+        json={"traceId": "my-trace-id"},
+        match_headers={"Authorization": "Bearer mock-ingestion-key"},
+        match_content=make_expected_body(
+            dict(
+                message="my-message",
+                traceId="my-trace-id",
+                timestamp=timestamp,
+                properties={
+                    "evaluations": [
+                        {
+                            "evaluatorExternalId": "my-evaluator",
+                            "id": str(test_evaluation_id),
+                            "score": 0.9,
+                            "metadata": None,
+                            "threshold": {"gte": 0.5},
+                        }
+                    ]
+                },
+            )
+        ),
+    )
+    tracer = AutoblocksTracer("mock-ingestion-key")
+    resp = tracer.send_event("my-message", **mock_input)
+    assert resp.trace_id == "my-trace-id"
+
+
+def test_tracer_failing_evaluation(httpx_mock):
+    class MyEvaluator(BaseEventEvaluator):
+        id = "my-evaluator"
+
+        def evaluate_event(self, event: TracerEvent) -> EventEvaluation:
+            raise Exception("Something terrible went wrong")
+
+    mock_input = {
+        "trace_id": "my-trace-id",
+        "timestamp": timestamp,
+        "properties": {},
+        "evaluators": [
+            MyEvaluator(),
+        ],
+    }
+    httpx_mock.add_response(
+        url=INGESTION_ENDPOINT,
+        method="POST",
+        status_code=200,
+        json={"traceId": "my-trace-id"},
+        match_headers={"Authorization": "Bearer mock-ingestion-key"},
+        match_content=make_expected_body(
+            dict(
+                message="my-message",
+                traceId="my-trace-id",
+                timestamp=timestamp,
+                properties={},
+            )
+        ),
+    )
+    tracer = AutoblocksTracer("mock-ingestion-key")
+    resp = tracer.send_event("my-message", **mock_input)
+    assert resp.trace_id == "my-trace-id"
```

Comment:
we need to make send_event fire and forget so that it doesn't block on evaluators

Reply:
Updated this and adjusted tests